### DISCUSSION TITLE
How do i restart POSTAL after setting up the spf record in the config file..... "postal restart" not working

### DISCUSSION THREAD

**Discussion by rolix202** [postalserver/postal#1667] [Help with using Postal] [answered]
I made changes to the spf record and DKIM record in my config file 
I tried restarting POSTAL but its not working

below is the message i receive

root@postal:~# postal restart
Usage: postal [command]

Running postal:

 * start - start Postal
 * stop - stop Postal
 * status - view current process status
 * logs [service] - view logs from services

Setup/upgrade tools:

 * initialize - create and load the DB schema
 * upgrade - upgrade the DB schema

Other tools:

 * version - show the current Postal version
 * make-user - create a new global admin user
 * default-dkim-record - display the default DKIM record
 * console - open an interactive console
 * test-app-smtp - send a test message through Postal
 * bootstrap hostname [path] - create a new set of config
 * bash [service] - access a bash console for a running process

how do i restart please
![postal error](https://user-images.githubusercontent.com/60781217/139347018-e45f33cd-0489-418d-b943-5e1dcfb5d837.png)

**Comment by willpower232** [comment, marked as answer]
I think you should be able to do `postal dc restart` but it would be nice to see this documented or implemented properly

**Reply by rolix202** [reply]
okay will try that out

**Reply by rolix202** [reply]
Thanks, it worked perfectly

Please how do i send mail to multiple mail address from my postal server

**Reply by willpower232** [reply]
Postal will send any email you give it so you can set multiple To's or CC's or BCC's when constructing the message you give it.

**Comment by pjv** [comment]
`postal stop`
`postal start`

not the most elegant thing in the world, but probably gets it done.

**Reply by willpower232** [reply]
indeed but `postal restart` is a command that used to exist so it would be nice to have some kind of documentation on its replacement

**Reply by pjv** [reply]
i added a restart command to v.2 postal in this PR: https://github.com/postalserver/install/pull/2
